extensions/ql-vscode/src/view/model-editor/ModelAlertsIndicator.tsx

```diff
@@ -36,6 +33,8 @@ export const ModelAlertsIndicator = ({
 
   return (
     <ModelAlertsButton
+      role="button"
+      aria-label="Model alerts"
```

Comment:
Added this to make the [tests](https://github.com/github/vscode-codeql/blob/eb1a5cfba382cbee9dddc0d46c74b592635a7788/extensions/ql-vscode/src/view/model-editor/__tests__/ModelAlertsIndicator.spec.tsx) pass and to make the button accessible 🤔 Not sure if this is the "normal" way of doing things? 🤷🏽